cmd/podman/untag.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/containers/libpod/cmd/podman/cliconfig"
+	"github.com/containers/libpod/pkg/adapter"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	untagCommand cliconfig.UntagValues
+
+	_untagCommand = &cobra.Command{
+		Use:   "untag [flags] IMAGE TAG [TAG...]",
+		Short: "Remove a name from a local image",
+		Long:  "Removes one or more names from a locally-stored image.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			untagCommand.InputArgs = args
+			untagCommand.GlobalFlags = MainGlobalOpts
+			untagCommand.Remote = remoteclient
+			return untag(&untagCommand)
+		},
+		Example: `podman unttag 0e3bbc2
+  podman untag imageID:latest otherImageName:latest
+  podman untag httpd myregistryhost:5000/fedora/httpd:v2`,
+	}
+)
+
+func init() {
+	untagCommand.Command = _untagCommand
+	untagCommand.SetHelpTemplate(HelpTemplate())
+	untagCommand.SetUsageTemplate(UsageTemplate())
+}
+
+func untag(c *cliconfig.UntagValues) error {
+	args := c.InputArgs
+
+	if len(args) == 0 {
+		return errors.Errorf("at least one image name needs to be specified")
+	}
+
+	runtime, err := adapter.GetRuntime(getContext(), &c.PodmanCommand)
+	if err != nil {
+		return errors.Wrapf(err, "could not create runtime")
+	}
+	defer runtime.DeferredShutdown(false)
+
+	newImage, err := runtime.NewImageFromLocal(args[0])
+	if err != nil {
+		return err
+	}
+
+	tags := args[1:]
+	if len(args) == 1 {
+		// Remove all tags if not explicitly specified
+		tags = newImage.Names()
+	}
+	logrus.Debugf("tags to be removed: %v", tags)
+
+	for _, tag := range tags {
+		msg := fmt.Sprintf("removing %q from %q", tag, newImage.InputName)
+		logrus.Debug(msg)
```

Comment:
Why not use `Debugf(..., x, y, z)`? Logs should be capitalized.

Reply:
I'm creating a dedicated `string` because I wanted to report the failing debug message as error some lines below :)

Reply:
Ah, thanks! The issue with using the same string is that by convention logs should start with upper and errors with lowercase.

Since we're returning the first error, I think we don't need to log it.

Reply:
Okay, I removed the log and just returned the lowercase error.